tests/acceptance/stepDefinitions/publicLinkContext.js

```diff
@@ -182,3 +182,21 @@ When('the user closes the public link details sidebar', function () {
     .filesList()
     .closeSidebar(100)
 })
+
+When('the user copies the public link named {string} of the file/folder/resource {string} using the webUI', async function (linkName, resource) {
+  await client.page.FilesPageElement
+    .filesList()
+    .closeSidebar(100)
+    .openPublicLinkDialog(resource)
+  return client.page.FilesPageElement.publicLinksDialog()
+    .clickCopyLinkURI(linkName)
+})
+
+When('the user navigates to the copied public link using the webUI', function () {
+  return client
+    .getClipBoardContent(function (url) {
+      setTimeout(function () {
```

Comment:
Please specify the reason behind using this pattern as comment.

Reply:
:rocket: